changelog-linker.yml

```
@@ -47,4 +48,5 @@ parameters:
         'migrations': 'shopsys/migrations'
         'monorepo-tools': 'shopsys/monorepo-tools'
         'microservice-product-search': 'shopsys/microservice-product-search'
-        'microservice-product-search-export': 'shopsys/microservice-product-search-export'
\ No newline at end of file
+        'microservice-product-search-export': 'shopsys/microservice-product-search-export'
+        'google-cloud-bundle': 'shopsys/google-cloud-bundle'
```

Diffs could be easier to manage if you end with an EOL:

```suggestion
        'google-cloud-bundle': 'shopsys/google-cloud-bundle'

```